src/ui/component/tag/tag.tsx

```diff
@@ -1,23 +1,38 @@
-import { type FC } from 'react'
+import type { MouseEvent } from 'react'
+import { useCallback, type FC } from 'react'
 import { Icon } from '@/ui/component/icon/icon'
 import './tag.scss'
 import { useTranslation } from 'react-i18next'
+import classNames from 'classnames'
 
 type TagProps = {
   tagName: string
-  onDelete?: () => void
+  classes?: { main?: string; name?: string; icon?: string }
+  onDelete?: (v: string) => void
```

Comment:
@oussama-aitbaziz-dev 
After more review, why this parameter has been added?
IMHO, it's not necessary.
You can see this implementation without parameter: https://github.com/okp4/dataverse-portal/blob/919d1b3d841de9669962585eeeb31955349fe67e/src/ui/view/tagsField/tagsField.tsx#L39

Reply:
@ccamel Suggested this above, I kinda agree with him that's the pattern I always found in react projects, every component should its own handlers

Reply:
@oussama-aitbaziz-dev 
@ccamel suggested to remove the currification, not to add a parameter.
Here, you gave back the `tag` value to the parent into the callback parameter that already knows the `value` (because it's the parent that injects the tag value as props)
So, I will remove this useless `v` parameter.